src/js/main.js

```diff
@@ -0,0 +1,35 @@
+let counter = sessionStorage.getItem('counter') || 0;
+let counterLine = document.getElementsByClassName('counterLine')[0];
+render(counterLine,  counter);
+
+document.addEventListener('DOMContentLoaded', function() {
+  document.addEventListener('click', (event)=> {
+    const element = event.target;
```

Comment:
where you use this `element`?

Reply:
It was for debugging. Just forgot to erase it...